lib/cc/engine/csslint/check_details.rb

```diff
@@ -0,0 +1,55 @@
+module CC
+  module Engine
+    class CSSlint
+      # https://github.com/CSSLint/csslint/wiki/Rules
+      class CheckDetails
```

Comment:
It seems like the major function of this class at present is to get the category for an issue.

What do you think about having a `category` class instead, and have it initialize with a check?

Then, in the `CSSLint` or an `Issue` class, we can have
```
def categories
  Category.new(check_name)
end

def remediation_points
   DEFAULT_REMEDIATION_POINTS
end
```